### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by manish-GH**
Created the avatar component

Preview: https://silly-euler-d8b50a.netlify.app/components/avatar/

### CODE REVIEW

**Review by Mohitk391 — APPROVED** [review]
Looks good..can be merged...

File: global-styles/general-layout.css [hunk 1]
```diff
@@ -33,3 +36,48 @@
...
+	justify-content: space-between;
+	flex-wrap: wrap;
+	background-color: var(--theme-bg-white);
+	border-radius: 1rem;
+	max-width: 1200px;
+	margin: 1rem 0;
+}
+.component-text {
+	max-width: 1200px;
+}
```
Comment (Sahil25061999) [inline global-styles/general-layout.css:56]: try using rem

File: global-styles/general-layout.css [hunk 2]
```diff
@@ -33,3 +36,48 @@
...
+	color: var(--theme-primary);
+	font-size: 1.6rem;
+}
+
+.code-snippet {
+	max-width: 1200px;
+}
+
+@media (min-width: 1800px) {
+	.flex-center-component {
```
Comment (Sahil25061999) [inline global-styles/general-layout.css:76]: use em in media query, rest of the code looks good to me

File: global-styles/general-layout.css (hunk 1 above)
Comment (manish-GH) [reply global-styles/general-layout.css:56]: nice catch, I'll do that

**Pull request merged**